x-pack/filebeat/input/entityanalytics/provider/okta/internal/okta/ratelimiter_test.go

```diff
@@ -166,4 +167,53 @@ func TestRateLimiter(t *testing.T) {
 			t.Errorf("unexpected rate following Update() (for fixed 120 reqs / 60 secs): %f", e.limiter.Limit())
 		}
 	})
+
+	t.Run("A concurrent rate limit should not set a new rate of zero", func(t *testing.T) {
+		const window = time.Minute
+		r := NewRateLimiter(window, nil)
+		const endpoint = "/foo"
+		url, err := url.Parse(endpoint)
+		if err != nil {
+			t.Errorf("unexpected error from url.Parse(): %v", err)
+		}
+		ctx := context.Background()
+		log := logp.L()
+		e := r.endpoint(endpoint)
```

Comment:
Linter is correct here, this is an ineffassign.

Reply:
The line is actually unnecessary. Removed.